.github/workflows/source/check_inputs.py

```diff
@@ -27,7 +27,7 @@
                     # skip lines related to other function arguments
                     # NOTE: update range call to reflect changes
                     #       in the interface of 'add_warpx_test'
-                    for _ in range(3):
+                    for _ in range(2):  # skip over: dims, numprocs
```

Comment:
Technically this would be
```suggestion
                    for _ in range(2):  # skip over: dims, nprocs
```
so that the inline comment matches the correct argument name. 

I can include this in one of the CTest follow-up PRs if we do not want to rerun all CI tests again on this PR.

Reply:
Thanks! Yes let's put in the follow-up to avoid rerunning the CI here.

Reply:
added in #5223